### PR TITLE
Implement function for retrieving a correlations results

### PR DESCRIPTION
**Pull request by ElRaptorus**
## What did you change?

Add necessary functions to the router and controller.

## How can others test the changes?

Implement the package and call the route.

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.)
- [x] I've merged the `develop` branch into my branch before finishing this PR.
- [x] I've **not added any other changes** than the ones described above.
- [x] I've mentioned all **PRs, which relate to this one**

### CODE REVIEW

File: src/consumer_api_controller.ts [hunk 1]
```diff
@@ -88,6 +89,16 @@ export class ConsumerApiController {
     response.status(this.httpCodeSuccessfulResponse).json(result);
   }
 
+  public async getCorrelationProcessModelResult(request: ConsumerRequest, response: Response): Promise<void> {
+    const correlationId: string = request.params.correlation_id;
+    const processModelKey: string = request.params.process_model_key;
+    const context: ConsumerContext = request.consumerContext;
+
+    const result: ICorrelationResult = await this.consumerApiService.getCorrelationProcessModelResult(context, correlationId, processModelKey);
```
Comment (cmg-dev) [inline src/consumer_api_controller.ts]: Ich würde das einfach `getProcessResult()` nennen.

File: src/consumer_api_controller.ts [hunk 2]
```diff
@@ -88,6 +89,16 @@ export class ConsumerApiController {
     response.status(this.httpCodeSuccessfulResponse).json(result);
   }
 
+  public async getCorrelationProcessModelResult(request: ConsumerRequest, response: Response): Promise<void> {
```
Comment (cmg-dev) [inline src/consumer_api_controller.ts]: Ich würde das einfach `getProcessResult()` nennen.

File: src/consumer_api_router.ts [hunk 3]
```diff
@@ -41,6 +41,7 @@ export class ConsumerApiRouter extends BaseRouter {
 
     this.router.get(restSettings.paths.processModels, wrap(controller.getProcessModels.bind(controller)));
     this.router.get(restSettings.paths.processModelByKey, wrap(controller.getProcessModelByKey.bind(controller)));
+    this.router.get(restSettings.paths.getCorrelationProcessModelResult, wrap(controller.getCorrelationProcessModelResult.bind(controller)));
```
Comment (cmg-dev) [inline src/consumer_api_router.ts]: Ich würde das einfach `getProcessResult()` nennen.

File: src/consumer_api_controller.ts (hunk 1 above)
Comment (ElRaptorus) [reply src/consumer_api_controller.ts]: Ich fand den Vorschlag `getProcessResultForCorrelation` besser. 🙂

**Comment by ElRaptorus** [comment]
@cmg-dev done

**Pull request merged**